.github/workflows/integration-tests.yaml

```
@@ -96,3 +96,15 @@ jobs:
           python3 -m pip install -e sdk/python; pytest sdk/python/test --log-cli-level=info --namespace=default
         env:
           GANG_SCHEDULER_NAME: ${{ matrix.gang-scheduler-name }}
+
+      - name: Collect volcano logs
+        if: ${{ failure() &&  env.GANG_SCHEDULER_NAME == 'volcano' }}
+        run: |
+          echo "dump volcano-scheduler logs..."
+          kubectl logs $(kubectl get po -n volcano-system | grep volcano-scheduler |cut -d" " -f1) -n volcano-system
+          echo "dump volcano-admission logs..."
+          kubectl logs $(kubectl get po -n volcano-system | grep volcano-admission |cut -d" " -f1 | grep -v init) -n volcano-system
+          echo "dump volcano-controllers logs..."
+          kubectl logs $(kubectl get po -n volcano-system | grep volcano-controllers |cut -d" " -f1) -n volcano-system
+          echo "dump podgroups description..."
+          kubectl describe podgroups.scheduling.volcano.sh -A
```

```suggestion
          echo "dump volcano-scheduler logs..."
          kubectl logs -n volcano-system -l app=volcano-scheduler
          echo "dump volcano-admission logs..."
          kubectl logs -n volcano-system -l app=volcano-admission
          echo "dump volcano-controllers logs..."
          kubectl logs -n volcano-system -l app=volcano-controller
          echo "dump podgroups description..."
          kubectl describe podgroups.scheduling.volcano.sh -A
```

Should we use label selectors?

---

good idear. I will also add `--tail=-1` to fetch all log instead of last 10 lines.

---

> . I will also add --tail=-1 to fetch all log instead of last 10 lines.

SGTM. Thanks!